watch_faces/BinarySystem/source/DataViews/StepsView.mc

```diff
@@ -6,11 +6,15 @@ using Toybox.WatchUi as Ui;
 
 class StepsView extends Ui.Drawable
 {
+    var stepsX = 0;
```

Comment:
since the step bar and steps text was linked i added these offset coordinates so the text can be moved relative to the bar position. you propose a different name?